a10_octavia/db/repositories.py

```diff
@@ -261,15 +261,19 @@ def get_vthunders_by_project_id(self, session, project_id):
         id_list = [model.id for model in model_list]
         return id_list
 
-    def get_vthunders_by_ip_address(self, session, ip_address):
+    def get_vthunders_by_ip_address(self, session, ip_address, vThunders=False):
```

Comment:
Instance variables should always be lower case

Reply:
Done